docker-compose.prod.yml

```diff
@@ -0,0 +1,35 @@
+services:
```

Comment:
we might not end up using this file, let's keep it just in case.

Reply:
I figured it was just a good reference for the services and the ports they use (and if you want to test it locally). Let's remember to delete it if it becomes out of date :P